src/App.js

```diff
@@ -1,24 +1,11 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import List_Products from './components/List_Products';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <List_Products />
```

Comment:
According to [React's best practices and naming conventions](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-pascal-case.md): `List_Products` is not a good component name. It should follow PascalCase format. Please, you might rename and fix it. 🙏

Reply:
Component name changed.

Reply:
I also changed the file name accordingly.

Reply:
I had to revert to previous file name because it affected deployment.